ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_attribute.py

```diff
@@ -1,3 +1,30 @@
 # global
 
 # local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="paddle.tensor.attribute.is_complex",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
```

Comment:
Please remember to always set the `available_dtypes` of the input arrays to "valid". In this case it might not make a difference in the test's results, but it's a matter of good practice.